lib/screens/main_menu/main_menu_page.dart

```diff
@@ -24,7 +24,7 @@ class MainMenuPage extends StatelessWidget {
               style: ElevatedButton.styleFrom(
                   primary: Theme.of(context).accentColor,
                   minimumSize:
-                      Size((MediaQuery.of(context).size.width) * 0.8, 100))),
+                      Size((MediaQuery.of(context).size.width) * 0.8, 64))),
```

Comment:
This still feels ugly and hard-codey, would be nice instead to do something properly generic using `MediaQuery.of(context)`.

I think in general we shouldn't need this ElevatedButtonTheme stuff - feels like we could achieve this instead using generic `Container` / `Expanded` type stuff. Will do a PR to show what I mean.